src/applications/dhp-connected-devices/components/ConnectedDevicesSection.jsx

```diff
@@ -46,7 +59,7 @@ export const ConnectedDevicesSection = ({
 };
 
 ConnectedDevicesSection.propTypes = {
-  connectedDevices: PropTypes.array.isRequired,
   failureAlert: PropTypes.bool.isRequired,
   successAlert: PropTypes.bool.isRequired,
+  connectedDevices: PropTypes.array,
```

Comment:
isRequired should accept empty arrays, too. It should reject undefined or non-array objects. This may have been removed during debugging, I'll add it back